### ISSUE TITLE
Loss for reg can't be reduced

### ISSUE THREAD

**Issue by changxinC** [wlguan/Stronger-yolo-pytorch#25]
I try to use KL loss in Mask R-CNN, but My training loss can't be reduced. Do you think there is something wrong with the following code?

The learning rate is 0.002, and I also try to use 0.0002. 
The learning rate decreases when training after 20000 and 40000 ,decay_gamma = 0.1, max_steps = 50000 .
Here is my KL code.
```
reg_target = self.bbox_transform_inv_xyxy(proposals[0].bbox, coor_target.bbox)
variance = variance[sampled_pos_inds_subset]
kl_loss = torch.exp(-variance)*smooth_l1_loss(box_regression[sampled_pos_inds_subset[:, None], map_inds],
                                              reg_target[sampled_pos_inds_subset], 1/9, KL=True) + variance/2
box_loss = kl_loss.mean(0).sum()*0.1
```
when start to train, the Lreg is about 0.02, but training after 50000 times it also nearly 0.02
Thank you!

**Comment by wlguan** [comment]
Sorry, no idea.
